Whack-a-mole/main.js

```diff
@@ -0,0 +1,78 @@
+const holes = document.querySelectorAll(".hole");
+const scoreBoard = document.querySelector(".score");
+const moles = document.querySelectorAll(".mole");
+const level1 = document.querySelector(".easy");
+const level2 = document.querySelector(".medium");
+const level3 = document.querySelector(".hard");
+const record = document.querySelector(".record");
+let lastHole;
+let timeUp = false;
+let score = 0;
+let rec = localStorage.getItem("totalScore");;
```

Comment:
initial value can be undefined so set it to 0 in this case

Reply:
done